idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -22,7 +22,7 @@ export default PageView.extend({
 
   events: {
     'click [data-hook=install]': 'install',
-    'click [data-hook=uninstall]': 'uninstall',
+    'click [data-hook=uninstall]': 'renderUninstallSurvey',
```

Comment:
I feel like it's pretty good practive to keep style hooks and js hooks separate.